core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/SegmentIndexFileStore.java

```diff
@@ -354,6 +364,22 @@ private MergedBlockIndex readMergeBlockIndex(ThriftReader thriftReader) throws I
     });
   }
 
+  /**
+   * List all the index files of the segment.
+   *
+   * @param segmentPath
+   * @return
+   */
+  public static CarbonFile[] getCarbonIndexFiles(String segmentPath, Configuration configuration) {
```

Comment:
Not required another method use an existing method to pass `configuration`

Reply:
ok